src/AppBundle/Security/BankIdService.php

```diff
@@ -46,7 +46,8 @@ public function getAccessToken($code)
 
         $accessToken = $client->get($url)->send()->getBody(true);
 
-        $this->getBankIdUser($accessToken);
```

Comment:
return $this->getBankIdUser($accessToken); have information from bank about user